crates/chain/src/local_chain.rs

```diff
@@ -173,6 +169,31 @@ impl LocalChain {
     pub fn heights(&self) -> BTreeSet<u32> {
         self.blocks.keys().cloned().collect()
     }
+
```

Comment:
same with the above we don't need it just expose the `BTreeMap`. Essentially LocalChain is just keeping a consistent BTreeMap so it only needs insert methods?